bufstream/iceberg-quickstart/cmd/producer/producer.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2020-2025 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"errors"
+	"github.com/brianvoe/gofakeit/v7"
+	demov1 "github.com/bufbuild/buf-quickstart/bufstream/iceberg-quickstart/gen/bufstream/demo/v1"
+
+	"github.com/bufbuild/buf-quickstart/bufstream/iceberg-quickstart/pkg/app"
```

Comment:
```suggestion
	"github.com/bufbuild/buf-examples/bufstream/iceberg-quickstart/pkg/app"
```

I'm assuming these need to be updated now that we've moved everything here

Reply:
Good catch. It'll work without the change, but it should be right.